### ISSUE TITLE
Symfony\Component\Locale\Locale class is deprecated

### ISSUE THREAD

**Issue by JHGitty** [lunetics/LocaleBundle#175]
```
The Symfony\Component\Locale\Locale class is deprecated since version 2.7, to be removed in Symfony 3.0. Use the methods provided by the \Symfony\Component\Intl\Intl class instead.

Called from require() at line 150
Called from DebugClassLoader::loadClass()
Called from spl_autoload_call() at line 82
Called from LocaleValidator::validate() at line 872
Called from RecursiveContextualValidator::validateInGroup() at line 711
Called from RecursiveContextualValidator::validateGenericNode() at line 138
Called from RecursiveContextualValidator::validate() at line 134
Called from RecursiveValidator::validate() at line 166
Called from RecursiveValidator::validateValue() at line 45
Called from MetaValidator::isAllowed() at line 62
Called from SessionLocaleGuesser::guessLocale() at line 120
Called from LocaleGuesserManager::runLocaleGuessing() at line 88
Called from LocaleListener::onKernelRequest()
Called from call_user_func() at line 61
Called from WrappedListener::__invoke()
Called from call_user_func() at line 1853
Called from EventDispatcher::doDispatch() at line 1771
Called from EventDispatcher::dispatch() at line 132
Called from TraceableEventDispatcher::dispatch() at line 120
Called from HttpKernel::handleRaw() at line 62
Called from HttpKernel::handle() at line 69
Called from ContainerAwareHttpKernel::handle() at line 190
Called from Kernel::handle() at line 30
Called from require() at line 36
```

**Comment by JHGitty** [comment]
I think this deprecation warning is about some other code but the same deprecated class is used here too:
https://github.com/lunetics/LocaleBundle/blob/6149a024be34afc7e7e9af362ad020037d0ab87f/Form/Extension/ChoiceList/LocaleChoiceList.php

**Comment by dbu** [comment]
would be awesome if you can do a pull request with a class_exists check
that uses the new class if available.

**Comment by JHGitty** [comment]
@dbu I don't have time for it. Anybody can do it :-)

**Comment by dbu** [comment]
well, same here. but thanks for reporting anyways.

**Comment by Ma27** [comment]
to be fixed in #179

**Comment by fkrauthan** [comment]
Any news on when you plan to release a SF3 compatible version?

**Comment by Ma27** [comment]
https://github.com/lunetics/LocaleBundle/releases/tag/2.4.5 ?

**Comment by fkrauthan** [comment]
Oh perfect. I just looked at the tickets and they are still all open^^

**Comment by dbu** [comment]
fixed in #179
